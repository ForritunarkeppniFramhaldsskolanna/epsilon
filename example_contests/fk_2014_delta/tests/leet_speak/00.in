9kyRf7 KQ6Sj3ufU2VSjhSsWD92je
