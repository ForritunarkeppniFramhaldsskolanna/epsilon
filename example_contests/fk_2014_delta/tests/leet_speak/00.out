9ky|2f7 |<(,)65j3uf|_|2|/5jh5s\/\/|)92je
